R/execution.R

```diff
@@ -745,6 +913,83 @@ writeJsonResultsToTable <- function(connectionDetails,
   )
 }
 
+.writeResultsToCsv <- function(checkResults,
+                           csvPath,
+                           columns = c("checkId", "FAILED", "PASSED", 
+                                       "IS_ERROR", "NOT_APPLICABLE",
+                                       "CHECK_NAME", "CHECK_DESCRIPTION",
+                                       "THRESHOLD_VALUE", "NOTES_VALUE",
+                                       "CHECK_LEVEL", "CATEGORY",
+                                       "SUBCATEGORY", "CONTEXT",
+                                       "CHECK_LEVEL", "CDM_TABLE_NAME",
+                                       "CDM_FIELD_NAME", "CONCEPT_ID",
+                                       "UNIT_CONCEPT_ID", "NUM_VIOLATED_ROWS",
+                                       "PCT_VIOLATED_ROWS", "NUM_DENOMINATOR_ROWS",
+                                       "EXECUTION_TIME", "NOT_APPLICABLE_REASON",
+                                       "ERROR", "QUERY_TEXT"),
+                           delimiter = "\t") {
```

Comment:
@MaximMoinat and @dmitryilyn let's talk about this, I see both sides but perhaps instead of exporting a csv we export a .txt which can be read into any application that takes in text files